ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4216,62 +4302,62 @@ AHU_US_MTV_E475_2: # Is actually a MAU
   - supply_fan_run_command
   - supply_fan_run_status
 
-AHU_US_MTV_FRC1_5: # Supply --> Discharge
```

Comment:
what happened here? this was changed but kept the previous id?

Reply:
They will all have the same ID if all that is changing is the name.